src/components/Options.tsx

```diff
@@ -142,6 +146,18 @@ export const Options = memo(function Options() {
             iconMap={sizeMetricIcons}
           />
         </fieldset>
+        <fieldset className="rounded-lg border p-2">
+          <legend className="card__title ml-1.5 justify-start gap-2">
+            <Icon path={mdiViewModule} size="1.25em" />
+            Chart depth
```

Comment:
Could we simplify this to just be "Depth", to align with the rest of the options?

Reply:
Good idea! Done! 🙂